### PR TITLE
Style campus qualification form

### LINKED ISSUE #146: style the campus-qualification form
Screen Shot 2019-04-01 at 10 22 11

### PR DESCRIPTION
**Pull request by boney-bun**
fix #146

### CODE REVIEW

File: django_project/ford3/static/css/campus_form.css [hunk 1]
```diff
@@ -1,3 +1,80 @@
...
+  background-position: 50% 50%;
+  background-color: #ffffff;
+  height: 5px;
+  border: none;
+  width: 61px;
+  height: 48px;
+}
+
+button.right-arrow-button {
+  background-image: url('../images/ui/remove_button.png');
```
Comment (pierrealixt) [inline django_project/ford3/static/css/campus_form.css]: let's call this image `right-arrow`

File: django_project/ford3/static/css/campus_form.css [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
 .selected {
-    background: red;
-}
\ No newline at end of file
+    background: #83cbd1;
+    color: #ffffff;
+}
+
+button.left-arrow-button {
+  background-image: url('../images/ui/add_button.svg');
```
Comment (pierrealixt) [inline django_project/ford3/static/css/campus_form.css]: let's call this image `left-arrow`

**Comment by boney-bun** [comment]
the PR in action:
ford3#146PR

**Review by pierrealixt — COMMENTED** [review]
I see the `add_button` image is a `svg` and the `remove_button` image is `png`.
Why?

File: django_project/ford3/templates/campus_form.html [hunk 3]
```diff
@@ -44,39 +49,38 @@ <h4 class="list-group-item-heading">
 
                         {% if wizard.steps.step1 == 4 %}
                             <div class='row'>
-                                <div class='col-md-5'>
-                                    <h5>All qualifications in {{ campus }} campus</h5>
+                                <div class='col-md-5 campus-all-qualifications'>
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html]: I'm not sure about this. `col-md-5` already set a width. We don't need another custom class with a specific width of 437px.

File: django_project/ford3/templates/campus_form.html [hunk 4]
```diff
@@ -44,39 +49,38 @@ <h4 class="list-group-item-heading">
...
+                                    <h5 class="text-center">All qualifications in {{ campus }} campus</h5>
                                     <ul id='campus-qualifications-list'>
                                     </ul>
                                 </div>
-                                <div class='col-md-2 text-center'>
+                                <div class='col-md-2 text-center' style="margin-top: 150px;">
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html]: Do not put css in the html.

File: django_project/ford3/templates/campus_form.html [hunk 5]
```diff
@@ -44,39 +49,38 @@ <h4 class="list-group-item-heading">
...
                                 </div>
                                 <div class='col-md-5'>
                                     <h5>All qualifications</h5>
-                                    <div class='row'>
-                                        <div class='col-md-10'>
-                                            <input type='text' data-action='search-qualif' />
-                                        </div>
-                                        <div class='col-md-2'>
+                                    <div class='row saqa-all-qualifications'>
+                                        <div class='col-md-12' style="box-shadow: 0 2px 2px rgba(0, 0, 0, 0.24), 0 0 2px rgba(0, 0, 0, 0.12);height: 40px">
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html]: same here, no css in HTML

File: django_project/ford3/templates/campus_form.html [hunk 6]
```diff
@@ -44,39 +49,38 @@ <h4 class="list-group-item-heading">
...
+                                            <input type='text' data-action='search-qualif' class="search-qualifications" value="Search Qualifications" style="width: 80%;"/>
                                             <button type='button'
                                                     data-action='clear-qualifs'>
                                                 clear
                                             </button>
+
                                         </div>
-                                        <div class='col-md-12'>
+                                        <div class='col-md-12' style="padding-right: 0px;">
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html]: I think there is a Bootstrap class to remove padding. Look at the documentation. Or create a custom class to not use css in html.

**Comment by boney-bun** [comment]
> I see the `add_button` image is a `svg` and the `remove_button` image is `png`.
> Why?

it's because when i imported the image in svg format from avocode, somehow the svg  missed the arrow's edge. while importing in png didn't have this problem.
do you encounter the same problem?

**Comment by pierrealixt** [comment]
@boney-bun did you make sure you imported all the parts of the image? 
I'll show you.

**Comment by boney-bun** [comment]
the PR:
ford3#146PRfix

**Comment by pierrealixt** [comment]
@boney-bun What happened to the form-step-bar?!
Screen Shot 2019-04-02 at 11 06 21

Please do not modify this part of the page, it was ok before. 
This PR is about styling the qualification form. 


Can you please make the http link obvious? (underline, different color?)

Can you make the `clear` button same height as the search bar?

**Comment by boney-bun** [comment]
@pierrealixt actually that's what i'm trying to fix.
the content isn't big enough to fit the screen.
but, it can adapt to screen sizes:
ford3#146StepBar

another problem regarding different screen sizes: `clear` button.

**Comment by boney-bun** [comment]
@pierrealixt this is the latest changes:
ford3#146PRFix2

File: django_project/ford3/static/css/wizard_form.css [hunk 7]
```diff
@@ -8,20 +8,26 @@
 }
```
Comment (pierrealixt) [inline django_project/ford3/static/css/wizard_form.css:9]: can you revert your changes in this file please?

File: django_project/ford3/templates/campus_form.html [hunk 8]
```diff
@@ -17,20 +17,25 @@
         <h2>{{ campus }} Campus Details</h2>
         <div class="row">
             <div class="col-lg-12">
-                <ul class="nav-wizard-steps nav nav-pills nav-fill">
-                    {% for form_name in form_name_list %}
-                        <li class="nav-item">
-                            <a class="nav-link
+                    <ul class="nav-wizard-steps nav nav-pills nav-fill">
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html:20]: can you revert your changes please?

File: django_project/ford3/templates/campus_form.html [hunk 9]
```diff
@@ -43,65 +48,59 @@ <h4 class="list-group-item-heading">
...
+                                        <div class='col-md-10 pl-1 pt-2 float-md-left border-bottom'>
+                                            <input type='text' data-action='search-qualif' class="search-qualifications" value="Search Qualifications" />
                                         </div>
-                                        <div class='col-md-2'>
-                                            <button type='button'
+                                        <div class="col-md-2 pt-2 float-md-right">
+                                            <button type='button' class="btn-danger btn-default"
                                                     data-action='clear-qualifs'>
-                                                clear
+                                                clear&nbsp;
```
Comment (pierrealixt) [inline django_project/ford3/templates/campus_form.html:78]: why adding `&nbsp`? 
Add `text-center` in the button classes.

**Pull request closed without merging**